docs/dev/vagrant.rst

```diff
@@ -31,9 +36,9 @@ This setup procedure does have some prerequisites of its own, which you will nee
 * `Python 2.7 <https://www.python.org/downloads/>`_
 * Python libraries ``fabric`` and ``fabtools`` (can be installed using pip, which comes with Python)
 
-If you are on a Linux system, it's likely that everything can be installed using a package manager on the command line, e.g. by running ``sudo apt-get install git virtualbox vagrant python2 python-pip && sudo pip install fabric fabtools``.
+If you are on a **Linux** system, it's likely that everything can be installed using a package manager on the command line, e.g. by running ``sudo apt-get install git virtualbox vagrant python2 python-pip && sudo pip install fabric fabtools``.
 
-If you are on a Windows system, it's easiest if you install the `PyCrypto binaries <http://www.voidspace.org.uk/python/modules.shtml#pycrypto>`_ before trying to install Fabric. In addition, you may need to run ``setx path "%path%;C:\Python27;C:\Python27\Scripts;"`` in order to put ``python``, ``pip`` and ``fab`` on your PATH. Finally, you may find that the Git Bash shell does not interact well with Fabric. The Windows Command Prompt works much better.
+If you are on a **Windows** system, it's easiest if you install the `PyCrypto binaries <http://www.voidspace.org.uk/python/modules.shtml#pycrypto>`_ before trying to install Fabric. In addition, you should modify the PATH environment variable from System -> Environment Variables by append ``C:\Python27;C:\Python27\Scripts;"`` in order to put ``python``, ``pip`` and ``fab`` on your PATH. (You can also run ``setx path "%path%;C:\Python27;C:\Python27\Scripts;"``, but this only takes effect for the current session.) You may also want to add the path to git's ``bin`` if you need an SSH client. Finally, you may find that the Git Bash shell does not interact well with Fabric. The Windows Command Prompt works much better.
```

Comment:
"In addition, you should modify the PATH environment variable (the system one, not the user one) in Control Panel -> System -> Environment Variables... by appending C:\Python27 and C:\Python27\Scripts. (You may also want to append the path to the "bin" folder in your git installation for an SSH client.) You will need to restart Command Prompt after making this change. To append, in Windows 10 you can edit Path and hit New to add new entries, and in previous versions of windows you should add them to the end of your path, separated by semicolons."